etcdctl/ctlv3/command/ep_command.go

```diff
@@ -95,15 +103,15 @@ func epHealthCommandFunc(cmd *cobra.Command, args []string) {
 	}
 
 	var wg sync.WaitGroup
-	errc := make(chan error, len(cfgs))
+	hch := make(chan epHealth, len(cfgs))
 	for _, cfg := range cfgs {
 		wg.Add(1)
 		go func(cfg *v3.Config) {
 			defer wg.Done()
 			ep := cfg.Endpoints[0]
 			cli, err := v3.New(*cfg)
 			if err != nil {
-				errc <- fmt.Errorf("%s is unhealthy: failed to connect: %v", ep, err)
```

Comment:
The output message when this error happens is going to be different due to this change.
@gyuho

Reply:
We can highlight this in CHANGELOG-3.3. Could you create another PR for this? Thanks!